src/wattsi.pas

```diff
@@ -1960,6 +1963,14 @@ function URLToID(const SpecURL: UTF8String; out ID: UTF8String): Boolean;
    {$ENDIF}
 end;
 
+procedure Inform(Message: UTF8String);
+begin
+   if Quiet then
```

Comment:
why not just "if (not Quiet) then Writeln()" ?

Also, style nits:
- put the expression in parentheses.
- spell it "Writeln" not "WriteLn".